Core/DotLanches.Application/Exceptions/ConflictException.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DotLanches.UseCases.Exceptions
```

Comment:
Acho que como estamos trabalhando com serviços, os projetos e namespaces deveriam refletir o serviço que estão.

Por exemplo, acredito que aqui deveria ser:
```suggestion
Dotlanche.Pedido.UseCases.Exceptions
```